docs/Principles/modeling-principles.md

```diff
@@ -0,0 +1,27 @@
+# Modeling Principles
+
+## Conforms to Schema
+The Monarch Biolink Specification is an implementation of the Biolink Model.
+The KG must be conformant with The Monarch Biolink Specification.
+
+## Node Normalization
+The final KG must have Nodes normalized to the canonical prefix for any given node type.
+The canonical prefix should be determined by The Monarch Biolink Model Specification.
+
+## Authoratative Source
+Providers of Associations are not the authoratative sources for the Nodes in general. Nodes should be ingested from their
+own authoratative source, seperate from edge ingests.
+
+## Genes and Proteins
+Genes and reference Proteins shall be treated as equivalent.
+When collapsing nodes give the Gene Id the priority, original_subject = UniProt Id.
```

Comment:
I don't understand how the equation at the end relates to the phrase at the beginning of this sentence (but that may just reflect my ignorance)

Reply:
We can hyperlink node properties, but overall most ingest devs will have the context. We could add something like 'devs must be familiar with biolink' as include link to bl docs